packages/ra-ui-materialui/src/index.js

```diff
@@ -1,4 +1,4 @@
-import Link from './Link';
+import { Link } from 'react-router-dom';
```

Comment:
I think we don't need to re-export the Link component. Ping @fzaninotto

Reply:
if it makes the upgrade easier... Just mark it as deprecated down where you export it.